index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="style.css">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name=" viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio Skeleton</title>
+</head>
+<body>
+  <div id="mobile-background">
+    <div id="Status-bar"></div>
+    
+    <header>
+      <a href="#">
+        <h2>Inyambo</h2>  
```

Comment:
- [ ] Kindly remove the text decorations on your header title so that it matches the design template you chose 👍🏼 

![image](https://user-images.githubusercontent.com/63473584/184975170-e6e0111f-75df-4bd5-b472-8a68332b5df9.png)